scripts/es.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
+const pkg = require('../package.json');
+
+require('@kbn/es').run({
+  version: pkg.version,
+  'source-path': path.resolve(__dirname, '../../elasticsearch'),
+  'base-path': path.resolve(__dirname, '../.es')
+});
```

Comment:
Need to make sure and catch errors here, log them, and set the process.exitCode = 1